doc/source/scipyoptdoc.py

```diff
@@ -105,7 +106,7 @@ def remove_arg(arg):
                 if opt_name in args:
                     continue
                 if j >= len(impl_args) - len(impl_defaults):
-                    options.append((opt_name, impl_defaults[len(impl_defaults) - (len(impl_args) - j)]))
+                    options.append((opt_name, impl_defaults[-len(impl_args) + j]))
```

Comment:
is the sign change on `j` here definitely correct?

Reply:
I think so. Simplifying to `x[len(x) - (len(y) - j)]`, let `len(x)` = `10`, `len(y)` = `5`, `j` = `3`.

`x[len(x) - (len(y) - j)]` = `x[10 - (5 - 3)]` = `x[10 - 2]` = `x[8]`.

`x[-len(y) + j]` = `x[-5 + 3]` = `x[-2]` = `x[8]`.

Reply:
You are right missed the second set of brackets hence the query around the sign